kolibri/core/auth/test/test_api.py

```diff
@@ -1501,24 +1507,24 @@ def test_facility_admin_can_check_is_pin_valid_unset_pin(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.data["extra_fields"]["pin_code"], None)
 
-        response = self.is_pin_valid({"pin_code": 1234})
+        response = self.is_pin_valid({"pin_code": "1234"})
         self.assertEqual(response.data["is_pin_valid"], False)
 
     def test_facility_admin_can_check_is_pin_valid_empty_pin_specified(self):
         self.client.login(username=self.superuser.username, password=DUMMY_PASSWORD)
-        response = self.update_pin({"pin_code": 1234})
-        response = self.is_pin_valid({"pin_code": None})
+        response = self.update_pin({"pin_code": "1234"})
```

Comment:
There are several spots in these tests where you assign `response = self.update_pin(...)` but then never end up using _that_ `response` variable but instead overwrite it shortly after. Just calling `self.update_pin` directly should be fine in any case where you're not asserting against that particular response.

Super minor and not a blocker.

Reply:
Thanks @nucleogenesis  for pointing this out. I have done cleanup on these as they are minor changes